src/main/kotlin/no/nav/syfo/infrastructure/kafka/VedtakStatusProducer.kt

```diff
@@ -10,32 +10,36 @@ import java.time.LocalDate
 import java.time.OffsetDateTime
 import java.util.*
 
-data class VedtakFattetRecord(
+data class VedtakStatusRecord(
```

Comment:
Det går an å gjøre dette annerledes på ulike måter, men kanskje dette er greit nok?

Som vi snakket om vil record'en ved ferdigbehandling bare skille seg fra den som kommer først (når vedtaket fattes)  ved at de to siste feltene har fått verdier.

Reply:
Funker jo dette, men jeg ville forventet et `status`-felt her (en enum som kan være FATTET eller FERDIGBEHANDLET). Og så kunne `veilederident` være veilederen som gjorde statusendringen og `createdAt` når statusendringen skjedde (kan evt renames til `generatedAt` e.l. hvis vi synes `createdAt` blir forvirrende)

Reply:
Ja, kan forsøke å lage et forslag.

Reply:
Ja, mulig jeg surrer litt nå, for jeg trodde egentlig vi landet på to topics: et med vedtaket, og ett litt "tynnere" med statusen. Men vi gikk kanskje bort fra det 🤔

Reply:
Ok, husker ikke hva vi landet på så vi kan sikkert ta en ny runde på det i morgen :sweat_smile: Men ser for meg én vedtak-hendelse/statusendring-topic vil funke greit for både dvh og våre apper.